### DISCUSSION TITLE
Is It Possible to compare only files of each group with each other

### DISCUSSION THREAD

**Discussion by sourabh-tiwari-gslab** [dodona-edu/dolos#1006] [Q&A] [answered]
Currently, How dolos works are when you send it a list of files. It compares each file with every file in the list. Is there a possibility that we could pass a group/cluster to the lib with files and they can only compare files of one group with each other? 

For Ex.

FileList = [file1.cs,file2.cs,file3.cs,file4.cs]

now is it possible to group this file as 
group1 = file1,file2
group2 = file3,file4

and pairs will be for file1-file2

and file3-file4 ?

* Currently we can achieve this by calling dolos repeatedly separately for each group files.

**Comment by rien** [comment, marked as answer]
Hi, this is currently not possible. We do have plans to add support for multiple exercises in the long term  (#338), but running Dolos separately is currently the best solution for this case.
